test/helpers/helpers.ts

```diff
@@ -37,3 +38,53 @@ export const scrollToElement = (element) => {
 export const waitForPresenceOfElement = (element) => {
   browser.wait(ExpectedConditions.presenceOf(element));
 };
+
+export const resetTestDataState = (username) => {
+  waitForOverlay('click-block-active');
+  clickBackButton();
+  // Logout
```

Comment:
No need for these comments, the code is basically supplies the information

Reply:
Removed